tools/topology/topology2/cavs-rt5682.conf

```diff
@@ -259,7 +268,7 @@ Object.PCM {
 		Object.Base.fe_dai."Speakers" {}
 
 		Object.PCM.pcm_caps."playback" {
-			name	"SSP1 Playback"
+			name	"$SPEAKER_PLAYBACK_PCM_CAPS"
```

Comment:
here it''s definitively pcm caps but is this really required to conflate caps and stream name?

Reply:
yes this is required to connect the PCM with the host component in the kernel.